homeassistant/bootstrap.py

```diff
@@ -325,6 +327,9 @@ def async_enable_logging(
     err_path_exists = os.path.isfile(err_log_path)
     err_dir = os.path.dirname(err_log_path)
 
+    fault_out = open(os.path.join(err_dir, FAULT_LOG_FILENAME), mode="w")
+    faulthandler.enable(fault_out)
```

Comment:
You're now enabling the fault handler twice?

Reply:
Yes, first before we know if the path for log is valid (just in case there is crash at startup) and second when we know is safe to write to the log.

Reply:
Is the 2nd enable overriding the first ?

Reply:
It looks like it's not automatically disabling when enabling a 2nd time, so you need to disable it first – or we just drop it from the first call.